Workbooks/UpdateCompliance/Feature updates/Feature updates.workbook

```diff
@@ -98,17 +98,17 @@
     {
       "type": 1,
       "content": {
-        "json": "The Feature Updates workbook provides information about the status of feature updates across all devices."
+        "json": "The Feature Updates workbook provides information about the status of feature updates across all devices.\r\n\r\n\r\n*Note: A device can be in multiple deployments. The data here reflects devices that are taking updates across **all deployments***."
       },
       "name": "IntroQualityUpdate"
     },
     {
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "// intialitze filters\r\nlet _SnapshotTime = datetime({_SnapshotTime});\r\n// Prepare the subtables\r\nlet UCClientUpdateStatus_Info = UCClientUpdateStatus | where TimeGenerated == _SnapshotTime;                               \r\nlet UCClient_Info = UCClient | where TimeGenerated == _SnapshotTime;\r\n\r\nUCClientUpdateStatus\r\n| join kind=innerunique UCClient_Info on AzureADDeviceId\r\n| where UpdateCategory == \"WindowsFeatureUpdate\"\r\n| extend DevicesTakingFeatureUpdate = \"Devices taking feature update\"\r\n| extend DevicesCount = \"Devices count\"\r\n| summarize TotalDevices=count() by DevicesTakingFeatureUpdate, DevicesCount\r\n\r\n",
+        "query": "let _SnapshotTime = datetime({_SnapshotTime});\r\nUCClient\r\n| where TimeGenerated == _SnapshotTime\r\n| where OSFeatureUpdateStatus == \"InService\"\r\n| extend DevicesTakingFeatureUpdate = \"Installed with in service feature update\"\r\n| extend DevicesCount = \"Devices count\"\r\n| extend ViewDetails = \"View Details\"\r\n| summarize TotalDevices=count() by DevicesTakingFeatureUpdate, DevicesCount, ViewDetails\r\n\r\n",
```

Comment:
Installed with in service feature update  ---> chnage to

Installed with an "inservice" feature update

Install with featureUpdate in service